doc/manual/src/installation/installing-binary.md

```diff
@@ -1,89 +1,112 @@
 
-# Single User Installation
-
-To explicitly select a single-user installation on your system:
+The installer can configured with various command line arguments and environment variables.
+To show available command line flags:
 
 ```console
-$ curl -L https://nixos.org/nix/install | sh -s -- --no-daemon
+$ curl -L https://nixos.org/nix/install | sh -s -- --help
 ```
 
-This will perform a single-user installation of Nix, meaning that `/nix`
-is owned by the invoking user. You can run this under your usual user
-account or root. The script will invoke `sudo` to create `/nix`
-if it doesn’t already exist. If you don’t have `sudo`, you should
-manually create `/nix` first as root, e.g.:
+To check what it does and how it can be customised further, [download and edit the second-stage installation script](#installing-from-a-binary-tarball).
+
+# Installing a pinned Nix version from a URL
+
+Version-specific installation URLs for all Nix versions since 1.11.16 can be found at [releases.nixos.org](https://releases.nixos.org/?prefix=nix/).
+The corresponding SHA-256 hash can be found in the directory for the given version.
+
+These installation scripts can be used the same as usual:
 
 ```console
-$ mkdir /nix
-$ chown alice /nix
+$ VERSION=2.19.2 curl -L https://releases.nixos.org/nix/nix-$VERSION/install | sh
 ```
 
-The install script will modify the first writable file from amongst
-`.bash_profile`, `.bash_login` and `.profile` to source
-`~/.nix-profile/etc/profile.d/nix.sh`. You can set the
-`NIX_INSTALLER_NO_MODIFY_PROFILE` environment variable before executing
-the install script to disable this behaviour.
-
 # Multi User Installation
 
-The multi-user Nix installation creates system users, and a system
-service for the Nix daemon.
+The multi-user Nix installation creates system users, and a system service for the Nix daemon.
+
+Supported systems:
 
-**Supported Systems**
 - Linux running systemd, with SELinux disabled
 - macOS
 
-You can instruct the installer to perform a multi-user installation on
-your system:
+You can explicitly instruct the installer to perform a multi-user installation on your system:
 
 ```console
 $ curl -L https://nixos.org/nix/install | sh -s -- --daemon
 ```
 
-The multi-user installation of Nix will create build users between the
-user IDs 30001 and 30032, and a group with the group ID 30000. You
-can run this under your usual user account or root. The script
-will invoke `sudo` as needed.
+You can run this under your usual user account or `root`.
+The script will invoke `sudo` as needed.
+
+# Single User Installation
+
+To explicitly select a single-user installation on your system:
+
+```console
+$ curl -L https://nixos.org/nix/install | sh -s -- --no-daemon
+```
+
+This will perform a single-user installation of Nix, meaning that `/nix` is owned by the invoking user.
+You can run this under your usual user account or `root`.
```

Comment:
Is this root bit true? I guess I've always taken the source at its word, here:

https://github.com/NixOS/nix/blob/188c803ddb5e63b243ddb84eba9b70e45475b7ea/scripts/install-nix-from-closure.sh#L135-L137

It looks like the language was changed from "not recommended" to "not supported" way back in 5d064e2.

Reply:
Also: I'm reading the previous sentence as trying to draw a distinction between multi- and single-user installs--but the assertion that we can invoke this as `root` seems to undercut or muddle that distinction.

Reply:
```suggestion
```

I was in fact also confused, but didn't bother to check. It was enough of a change as it is.